File: api/src/main/java/fr/huiitre/tools/modules/riot/valorant/application/catalog/view/ValorantStoreView.java
package fr.huiitre.tools.modules.riot.valorant.application.catalog.view;

import java.util.List;

public record ValorantStoreView(
    List<ValorantStoreOffer> offers,
    long remainingSeconds,
    List<ValorantStoreBundle> bundles,
    ValorantNightMarket nightMarket
) {}

File: api/Modules/Riot/Valorant/Application/Catalog/Views/ValorantStoreView.cs
namespace Tools.Api.Modules.Riot.Valorant.Application.Catalog.Views;

public sealed record ValorantStoreView(
    List<ValorantStoreOffer> Offers,
    long RemainingSeconds,
    List<ValorantStoreBundle> Bundles,
    // Absent la plupart du temps : le marché nocturne n'ouvre que par périodes.
    ValorantNightMarket? NightMarket
);